test_runner/regress/test_tenant_conf.py

```diff
@@ -69,7 +69,18 @@ def test_tenant_config(neon_env_builder: NeonEnvBuilder):
                     "image_creation_threshold": 3,
                     "pitr_interval": 604800,  # 7 days
                 }.items()
-            )
+            ), f"Unexpected res: {res}"
+    default_tenant_config = http_client.tenant_config(tenant_id=env.initial_tenant)
+    assert not default_tenant_config.tenant_specific_config, "Should have no specific settings yet"
+    default_resulting_config = default_tenant_config.resulting_config
+    assert default_resulting_config["checkpoint_distance"] == 10000
+    assert default_resulting_config["compaction_target_size"] == 1048576
+    assert default_resulting_config["compaction_period"] == "20s"
+    assert default_resulting_config["compaction_threshold"] == 10
+    assert default_resulting_config["gc_horizon"] == 67108864
+    assert default_resulting_config["gc_period"] == "1h"
+    assert default_resulting_config["image_creation_threshold"] == 3
+    assert default_resulting_config["pitr_interval"] == "7days"
```

Comment:
If only we'd have a type which could equal to psycopg's postgres period/interval AND humantime, we use the same dict to assert both.. Is it possible to make one type? I assume there's already something like that going on, but it might just be psycopg's type supporting `== int (seconds)` or `== float (seconds)`.

Reply:
Not sure I am aware of such. 
At least now with the hardcoded test, we're testing that our humatime serialization format is unchanged 🙂

Reply:
Giving up it is. These tests should be split before adding anything new to them next time around..